cp22/hpi-fun2/hexhex/executables/solution.py

```diff
@@ -0,0 +1,50 @@
+import io
+import os
+
+from regex import X
+
+input = io.BytesIO(os.read(0, os.fstat(0).st_size)).readline
```

Comment:
This should not be necessary to solve the problem in Python. Please remove. (I checked, it is still fast enough on the judge).

Reply:
Yep, IO optimization doesn't matter since the input is really small (at most 101 integers). Removed.